GUI/Main.cs

```diff
@@ -187,6 +187,21 @@ public Main(string[] cmdlineArgs, GUIUser User, bool showConsole)
                     Repo.default_ckan_repo.ToString()
                 );
 
+            // Check if there is any other instances already running.
+            // This is not entirely necessary, but we can show a nicer error message this way.
+            try
+            {
+                #pragma warning disable 219
```

Comment:
This may be one for @mgsdk (who authored the change). I suspect calling `RegistryManager.Instance(CurrentInstance).registry` without an assignment generates a warning and/or requires an assignment, but I honestly haven't tested that.